kebechet/managers/version/release_triggers.py

```diff
@@ -94,7 +94,12 @@ def adjust_version_in_sources(
     ) -> List[Tuple[str, str, str]]:
         """Walk through the directory structure and try to adjust version identifier in sources."""
         adjusted = []
-        for root, _, files in os.walk("./"):
+        for root, dirs, files in os.walk("./"):
+            dirs[:] = [
+                d for d in dirs if not d[0] == "."
+            ]  # remove hidden dirs (done in place to remove from walk)
+            files = [f for f in files if not f[0] == "."]  # remove hidden files
```

Comment:
As os.walk would explicity walk over each file sub-directory after sub-directory
maybe we could just check the root ?
```suggestion
     if not root.startswith("./.")             
```
wdyt about this ?

Reply:
The in place redefinition of dirs keeps walk from iterating over hidden directories and finding those files

Reply:
thanks.